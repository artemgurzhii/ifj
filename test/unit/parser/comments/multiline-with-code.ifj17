' foo
' bar
' baz

dim foo as string

' foo
' bar
' baz

dim bar as integer

' foo
' bar
' baz
