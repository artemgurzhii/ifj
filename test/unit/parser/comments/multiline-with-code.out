(dim
  (decl
    (id foo)
    : (id string)))

(dim
  (decl
    (id bar)
    : (id integer)))
